pkgs/tools/typesetting/tex/blahtexml/default.nix

```diff
@@ -0,0 +1,44 @@
+{ fetchFromGitHub, lib, stdenv, libiconv, texlive, xercesc }:
+
+stdenv.mkDerivation {
+  pname = "blahtexml";
+  version = "0.9+unstable=20200516";
```

Comment:
```suggestion
  version = "unstable-2020-05-16";
```

Reply:
I have updated it so that it follows the current version https://github.com/NixOS/rfcs/pull/107 (i.e. use `+date=YYYY-MM-DD`). I have seen comments about naked `unstable-` causing problems... ?

Reply:
Hmm okay. The RFC is not accepted yet, but I guess this is fine as well. In that case, add the dashes for the date, as shown in the RFC.

Reply:
Thanks! Yes, I have updated the version number with `date` and dashes.